.github/CONTRIBUTING.md

```diff
@@ -16,7 +16,9 @@ One important thing distinction is that we are building the Polkadot Runtime Env
 
 ## Contribution Steps
 
-**1. Set up gossamer following the instructions our [README](https://github.com/ChainSafe/gossamer/blob/development/README.md).**
+**1. Set up gossamer following the instructions our [README](https://github.com/ChainSafe/gossamer/blob/development/README.md).
+     Please note you should be in ~/go/src/github.com/ChainSafe/gossamer in order to run the go get -u github.com/ChainSafe/gossamer. 
```

Comment:
not sure if this makes sense, you should be able to run `go get -u github.com/ChainSafe/gossamer` anywhere and it creates the `gossamer` directory for you

Reply:
Had this issue on 2 different machines, just tried it again. Would you accept if rephrased to:  If you are getting a "No Go files.." error, please make sure you are in ~/go/src/github.com/ChainSafe/gossamer. 

![Screenshot from 2020-07-24 09-00-44](https://user-images.githubusercontent.com/12500447/88411720-48949000-cd8d-11ea-84b9-367bf64d47ce.png)

Reply:
since it's not an error, I would say change this to something like in the README:

> You may encounter a `package github.com/ChainSafe/gossamer: no Go files in ...` message when doing `go get`. This is not an error, since there are no go files in the project root.

however, now step 1 and 3 are duplicated, so can you remove step 1, then add the above message under the commands in step 3? you will also have to re-number the steps after